src/Entity/Admin.php

```diff
@@ -50,6 +50,13 @@ public function getUsername(): string
         return (string) $this->username;
     }
 
+    public function setUsername(?string $username): self
```

Comment:
You don't need a getter, the username is public :) . Moreover, I guess it would make sense to put the username in the constructor, as creating an admin without username would be invalid.

Reply:
Ah my bad, the constructor bit won't work with EasyAdmin. The setter is still not required though.

Reply:
Good point - updated.